### DISCUSSION TITLE
申请下退款

### DISCUSSION THREAD

**Discussion by suduishan** [getlantern/lantern-forum-cn#986] [General]
新买的手机，立即购买了蓝灯，但是一直没法升级系统。
申请下退款，谢谢！
![Screenshot_20241023-175502](https://github.com/user-attachments/assets/720ba844-78ec-4602-a4ec-587ffefee5c6)

**Comment by MaxMKMax** [comment]
@suduishan 您好. 请您先别退款. 我帮您申请手动升级. 请您提供我您的账号邮箱.我已经帮您提交问题上去了. 等待处理. 非常抱歉啊. 我们会尽快给您解决问题的.

**Reply by suduishan** [reply]
实在抱歉，我已经购买别的了。
我邮件反馈过多次了。

**Reply by suduishan** [reply]
麻烦处理下退款，谢谢您。
suduishan@outlook.com

**Reply by suduishan** [reply]
> @suduishan 您好. 请您先别退款. 我帮您申请手动升级. 请您提供我您的账号邮箱.我已经帮您提交问题上去了. 等待处理. 非常抱歉啊. 我们会尽快给您解决问题的.

你好，请问什么时候能退款？

**Reply by suduishan** [reply]
> @suduishan 您好. 请您先别退款. 我帮您申请手动升级. 请您提供我您的账号邮箱.我已经帮您提交问题上去了. 等待处理. 非常抱歉啊. 我们会尽快给您解决问题的.

请问什么时候能处理完退款？

**Comment by suduishan** [comment]
请问退款流程处理怎么样了？

**Reply by MaxMKMax** [reply]
@suduishan 您好. 已经帮您提交退款申请了. 请您等待回复.

**Reply by liuh23** [reply]
我的退款呢，怎么一直不给我回复？？？
https://github.com/getlantern/lantern-forum-cn/discussions/973

**Reply by liuh23** [reply]
这是答应我退款的链接https://github.com/getlantern/lantern-forum-cn/discussions/973

**Reply by suduishan** [reply]
> @suduishan 您好. 已经帮您提交退款申请了. 请您等待回复.

你好，怎么这么久了一直没有退款啊？

**Reply by MaxMKMax** [reply]
@suduishan 我帮您问一下.

**Reply by suduishan** [reply]
请问退款呢？这么久了一直没有反馈呢？

**Reply by suduishan** [reply]
> @suduishan 我帮您问一下.

你好，请问为什么这么久一直没有处理呢？

**Comment by MaxMKMax** [comment]
@suduishan 您好. 我刚刚给您开个case给支付部门处理了. 请您之后查看邮箱. 谢谢.
